QAT/centos-7.4/media/dev/Dockerfile

```
@@ -67,9 +73,10 @@ RUN wget -O - ${QATZIP_REPO} | tar xz && \
 ARG QAT_OPENSSL_VER=OpenSSL_1_1_1c
 ARG QAT_OPENSSL_REPO=https://github.com/openssl/openssl/archive/${QAT_OPENSSL_VER}.tar.gz
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN wget -O - ${QAT_OPENSSL_REPO} | tar xz && mv openssl-${QAT_OPENSSL_VER} openssl && \
     cd openssl && \
-    ./config --prefix=/opt/openssl --openssldir=/opt/openssl -Wl,-rpath,\${LIBRPATH} && \
+    ./config --prefix=/opt/openssl --openssldir=/opt/openssl -Wl,-rpath,${LIBRPATH} && \
```

```suggestion
    ./config --prefix=/opt/openssl --openssldir=/opt/openssl "-Wl,-rpath,${LIBRPATH}" && \
```

did shellcheck not ask you to quote this?